.gitignore

```diff
@@ -1 +1,3 @@
 *.swp
+**/build
```

Comment:
this file should be removed from the commit

Reply:
Fixed